independent-projects/tools/common/src/main/java/io/quarkus/generators/rest/BasicRestProjectGenerator.java

```diff
@@ -171,10 +171,10 @@ private void createReadme() throws IOException {
             BuildTool buildTool = getBuildTool();
             switch (buildTool) {
                 case MAVEN:
-                    generate("templates/README.maven.ftl", context, readme, "read me");
```

Comment:
Will this whitespace change break the formatter?

Reply:
I ran the full build before submitting, so I guess it won't

Reply:
Apparently not because it is in independent-projects

Reply:
I executed a sample `mvn clean install` and the source wasn't changed so I guess the formatter didn't have any issues